schema-loader/src/main/java/com/scalar/db/schemaloader/command/StorageSpecificCommandBase.java

```diff
@@ -26,11 +26,21 @@
       defaultValue = "false")
   private boolean deleteTables;
 
+  @Option(
+      names = {"--prefix"},
+      description = "Namespace prefix",
+      hidden = true)
+  protected String prefix;
```

Comment:
Let's have this parameter only for CosmosDB.

Reply:
Thanks for the suggestion, I updated it in [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/19bcd1a164334f2073f0a94d5a52bee4a2530ac2)